flexmeasures/ui/templates/crud/account.html

```diff
@@ -1,183 +1,345 @@
+                    <button
+                      class="btn btn-sm btn-responsive btn-success create-button"
+                      type="submit"
+                    >
+                      Create new asset
+                    </button>
+                  </form>
+                  {% endif %}
+                </th>
+                <th class="d-none">URL</th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for asset in assets %}
+              <tr>
+                <td>
+                  <i
+                    class="{{ asset.generic_asset_type.name | asset_icon }} left-icon"
+                    >{{ asset.name }}</i
+                  >
+                </td>
+                <td>
+                  {% if asset.latitude and asset.longitude %} LAT: {{
+                  "{:,.4f}".format( asset.latitude ) }} LONG: {{
+                  "{:,.4f}".format( asset.longitude ) }} {% endif %}
+                </td>
+                <td>{{ asset.id }}</td>
+                <td>
+                  {% if asset.owner %} {{ asset.owner.name }} {% else %} PUBLIC
+                  {% endif %}
+                </td>
+                <td>{{ asset.sensors | length }}</td>
+                <td>
+                  <a href="/assets/{{ asset.id }}/status">
+                    <button type="button" class="btn">Status</button>
+                  </a>
+                </td>
+                <td class="text-right"></td>
+                <td class="d-none">/assets/{{ asset.id }}</td>
+              </tr>
+              {% endfor %}
+            </tbody>
+          </table>
         </div>
+      </div>
     </div>
+    <div class="col-md-2"></div>
+  </div>
 </div>
-<div class="col-md-2"></div>
 {% block paginate_tables_script %} {{ super() }} {% endblock %}
+<script type="text/javascript">
+  const basePath = window.location.origin;
+
+  // Select the specific form with ID 'editaccount'
+  document
+    .querySelector("#editaccount")
+    .addEventListener("submit", function (event) {
+      event.preventDefault(); // Prevent the default form submission
+
+      // Collect form data
+      const formData = {
+        name: document.getElementById("name").value,
+        primary_color: document.getElementById("primary_color").value,
+        secondary_color: document.getElementById("secondary_color").value,
+        consultancy_account_id: document.getElementById("consultant_account_id")
+          .value,
+        logo_url: document.getElementById("logo_url").value,
+      };
 
+      // Send data to the API using a PATCH request
+      fetch(basePath + "/api/v3_0/accounts/" + "{{account.id}}", {
+        method: "PATCH",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(formData),
+      })
+        .then((response) => response.json())
+        .then((data) => {
+          if (data.status == 200) {
+            alert("Form data updated successfully!");
```

Comment:
I planned to use Bootstrap 5's popup component. It took me a while to get it working because Bootstrap didn't "activate" it out of the box, but I eventually figured it out, and it's functioning well now. The UX might not be perfect though